physx/src/transform.rs

```diff
@@ -48,70 +40,45 @@ impl Into<PxQuat> for PxQuatWrap {
     }
 }
 
-struct PxTransformWrap(PxTransform);
-impl From<Isometry3<f32>> for PxTransformWrap {
-    /// Convert from an nalgebra isometry into a PxTransform
-    fn from(other: Isometry3<f32>) -> Self {
-        let quat: glm::Quat = glm::convert(other.rotation);
-        let quaternion: PxQuat = PxQuatWrap::from(quat).into();
-        let trans = other.translation;
-        PxTransformWrap(unsafe { PxTransform_new_4(trans.x, trans.y, trans.z, quaternion) })
-    }
+/// Convert a Mat4 to a PxTransform
+pub fn gl_to_px_tf(trans: Mat4) -> PxTransform {
+    gl_to_px_tf_ref(&trans)
 }
 
-impl Into<PxTransform> for PxTransformWrap {
-    /// Unwrap helper
-    fn into(self) -> PxTransform {
-        self.0
-    }
+/// Convert a reference to a Mat4 to a PxTransform
+pub fn gl_to_px_tf_ref(trans: &Mat4) -> PxTransform {
+    let quat: PxQuatWrap = Quat::from_rotation_mat4(&trans).into();
+    let trans = trans.w_axis().truncate();
```

Comment:
Might be better to do `let (x, y, z, _) = trans.w_axis();` rather than accessing each component separately.